docs/components/arm/eva.md

```diff
@@ -12,7 +12,9 @@ aliases:
 # SMEs: Bucket, Motion
 ---
 
-Configure an `eva` arm to integrate a [Automata Eva](https://automata.tech/products/hardware/about-eva/) robotic arm into your machine:
+Configure an `eva` arm to integrate a [Automata Eva](https://automata.tech/products/hardware/about-eva/) robotic arm into your machine.
+Make sure to first physically assemble the Eva, connect it to your machine's computer, and turn it on.
```

Comment:
```suggestion
Make sure to first connect the arm to your machine's computer and turn it on.
```
I'm not sure how much assembly is required on various arms. Also, for some models like DOFBOT, I'm not sure whether the manufacturer actually calls it "the DOFBOT" or if it would be "the DOFBOT arm," so it's probably best to keep it generic and simplify.

Reply:
For posterity to surface offline convo: I did check with Ray and it sounds like off-the-shelf arms tend to ship fully assembled.